<?xml version='1.0' encoding='UTF-8'?>
<robot name="sphere_robot">
  <link name="sphere_link">
    <gravity>0</gravity>
    <visual>
      <geometry>
        <sphere radius="0.0"/>
      </geometry>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
    </visual>
    <collision>
      <geometry>
        <sphere radius="0.0"/>
      </geometry>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
    </collision>
    <inertial>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
      <mass value="1.0"/>
      <inertia ixx="1.0" ixy="0.0" ixz="0.0" iyy="100.0" iyz="0.0" izz="1.0"/>
    </inertial>
  </link>
  <gazebo reference="sphere_link">
    <material>Gazebo/Orange</material>
    <turnGravityOff>1</turnGravityOff>
  </gazebo>
</robot>